src/read_data.cpp

```diff
@@ -1963,7 +1963,7 @@ void ReadData::parse_keyword(int first)
     }
     while (eof == 0 && done == 0) {
       int blank = strspn(line," \t\n\r");
-      if ((blank == (int)strlen(line)) || (line[blank] == '#')) {
```

Comment:
please don't undo changes in the LAMMPS code, that have no relevance to your change.

Reply:
hmm, for some reason that didn't update when I pulled from the lammps repository; not intentional.